backend/src/config/neo4j/createNeo4jDriver.ts

```diff
@@ -6,22 +6,57 @@ import {
   Driver,
 } from 'neo4j-driver';
 
-/**
- * Adapted from @nest-neo4j's createDriver
- * @param config
- * @param neo4jConfig
- */
-export const createNeo4jDriver = async (
+function delay(timeout: number): Promise<void> {
+  if (timeout <= 0) {
+    return Promise.resolve();
+  }
+
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, timeout);
+  });
+}
+
+async function tryConnect(driver: Driver): Promise<boolean> {
+  let session;
+  try {
+    await driver.verifyConnectivity();
+    session = driver.session();
+    await session.run('MATCH (n) RETURN n LIMIT 1');
+    return true;
+  } catch (e) {
+    return false;
+  } finally {
+    await session?.close();
+  }
+}
+
+async function createDriverWithRetry(
```

Comment:
Documentation was removed
```suggestion
/**
 * Adapted from @nest-neo4j's createDriver
 * @param config
 * @param neo4jConfig
 */
async function createDriverWithRetry(
```

Reply:
It was not removed, the documentation is on the const export of the function below.